File: sic-api/Entities/Ex/ExExample.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using sic_api.Attributes;
using sic_api.Model.Storage;

namespace sic_api.Entities.Ex;

[Table("ex_example")]
public class ExExample : BaseEntity
{
    [Required]
    [MaxLength(50)]
    [Column("example_code")]
    public string ExampleCode { get; set; } = default!;

    [Required]
    [Column("message_en")]
    public string MessageEn { get; set; } = default!;

    [Required]
    [Column("message_local")]
    public string MessageLocal { get; set; } = default!;

    [Required]
    [Column("start_date")]
    public DateTime StartDate { get; set; }

    [Required]
    [Column("end_date")]
    public DateTime EndDate { get; set; }

    [Column("start_time")]
    public string? StartTime { get; set; }

    [Column("end_time")]
    public string? EndTime { get; set; }

    [Column("is_accept")]
    public string? IsAccept { get; set; }

    [Column("color")]
    public string? Color { get; set; }

    [Column("country_code")]
    public string? CountryCode { get; set; }

    [Required]
    [Column("total")]
    public long Total { get; set; }

    [Column("upload_group_id")]
    [Storage("UploadGroupData")]
    public Guid? UploadGroupId { get; set; }

    [Required]
    [Column("is_active")]
    public bool IsActive { get; set; } = false;

    [NotMapped]
    public List<StorageUploadReference>? UploadGroupData { get; set; }
}

File: sic-spring/sic/src/main/java/com/softinter/sicapi/entity/ex/ExExample.java
package com.softinter.sicapi.entity.ex;

import com.softinter.sicapi.entity.base.BaseEntity;
import jakarta.persistence.*;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.time.Instant;
import java.time.LocalDate;
import java.time.LocalTime;
import java.util.List;
import java.util.UUID;

@Entity
@Table(name = "ex_example")
@Data
@EqualsAndHashCode(callSuper = true)
@NoArgsConstructor
public class ExExample extends BaseEntity {

     @Column(name = "example_code", length = 100)
    private String exampleCode;

    @Column(name = "message_en", length = 500)
    private String messageEn;

    @Column(name = "message_local", length = 500)
    private String messageLocal;

    @Column(name = "start_date")
    private LocalDate startDate;

    @Column(name = "end_date")
    private LocalDate endDate;

    @Column(name = "start_time")
    private LocalTime startTime;

    @Column(name = "end_time")
    private LocalTime endTime;

    @Column(name = "is_accept")
    private Boolean isAccept = false;

    @Column(name = "color", length = 50)
    private String color;

    @Column(name = "country_code", length = 10)
    private String countryCode;

    @Column(name = "total")
    private BigDecimal total;

    @Column(name = "upload_group_id")
    private UUID uploadGroupId;

    @Column(name = "is_active")
    private Boolean isActive = true;

    @Transient
    private List<StorageUploadReference> uploadGroupData;
}
